woocommerce/woocommerce-ios
# Storage: add fees to orders

This is the second step to add support for Fees to the Order Details page. (#3084). The first step is here: #3452 

This PR implements support for Fees in Storage. Support for Fees in Yosemite will be added in a subsequent PR.

## Changes
* Added a new version to coredata model
* Added a new entity: `OrderFeeLine` and new relationships to `Order`
* Added method to retrieve Fees to StorageType+Extensions

## How to test
* 👀 on the code. I haven't done this in a while, so I presume I have missed something.
* Coming from develop, check out the branch, build and run. There shouldn't be any coredata errors or warnings.
* CI should be ✅ 

Update release notes:
